src/models/minecraft_types.rs

```diff
@@ -10,3 +12,34 @@ pub struct ChunkSection {
     pub block_light: Vec<u64>, //2048 bytes (all 1s)
     pub sky_light: Vec<u64>,   //2048 bytes (all 1s)
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Version {
+    pub name: String,
+    pub protocol: u16,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct SamplePlayer {
+    pub name: String,
+    pub id: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Players {
+    pub max: u16,
+    pub online: u16,
+    pub sample: Vec<SamplePlayer>,
+}
```

Comment:
Let's append something to the name here to make it clear that this is only for the status response and doesn't related to how a player is stored. I think the module name here isn't enough to make it clear otherwise